File: python/Ejercicio46.py
import math

G = 6.674e-11
masa_planeta = float(input("Ingrese la masa del planeta (kg): "))
radio_planeta = float(input("Ingrese el radio del planeta (m): "))

velocidad_escape = math.sqrt(2 * G * masa_planeta / radio_planeta)

print("La velocidad de escape es:", velocidad_escape, "m/s")

File: java/Ejercicio46.java
import java.util.Scanner;


public class Ejercicio46 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        

        final double G = 6.674e-11;

        System.out.print("Ingrese la masa del planeta (kg): ");
        double masaPlaneta = sc.nextDouble();

        System.out.print("Ingrese el radio del planeta (m): ");
        double radioPlaneta = sc.nextDouble();

        double velocidadEscape = Math.sqrt(2 * G * masaPlaneta / radioPlaneta);

        System.out.println("La velocidad de escape es: " + velocidadEscape + " m/s");

        sc.close();
    }
}